### ISSUE TITLE
How do I provide an argument to use an image target rather than a text prompt?

### ISSUE THREAD

**Issue by teclados078** [threedle/text2mesh#25]
I removed --prompt from the settings of run_shoe.sh and set values for --no_prompt and --image, and the quality of results were really bad. Details are as follows:

**_case1_**
* run_shoe.sh
`python main.py --run branch --obj_path data/source_meshes/shoe.obj --output_dir results/demo/shoe/texture/brick --no_prompt --image data/target_texture/brick_texture.jpg --sigma 5.0  --clamp tanh --n_normaugs 4 --n_augs 1 --normmincrop 0.1 --normmaxcrop 0.1 --geoloss --colordepth 2 --normdepth 2   --frontview --frontview_std 4 --clipavg view --lr_decay 0.9 --clamp tanh --normclamp tanh  --maxcrop 1.0 --save_render --seed 11 --n_iter 1500  --learning_rate 0.0005 --normal_learning_rate 0.0005 --background 1 1 1 --frontview_center 0.5 0.6283
`

* brick_texture.jpg
![image](https://user-images.githubusercontent.com/111476605/190899030-ab27b971-4eed-4d68-9d49-c311f39518ff.png)

* result
![image](https://user-images.githubusercontent.com/111476605/190899109-915c0c8e-2ab7-4c77-84f6-b0c03df7e7f4.png)

**_case2_**
* run_shoe.sh
`python main.py --run branch --obj_path data/source_meshes/shoe.obj --output_dir results/demo/shoe/texture2/cactus --no_prompt --image data/target_texture/cactus_texture.jpg --sigma 5.0  --clamp tanh --n_normaugs 4 --n_augs 1 --normmincrop 0.1 --normmaxcrop 0.1 --geoloss --colordepth 2 --normdepth 2   --frontview --frontview_std 4 --clipavg view --lr_decay 0.9 --clamp tanh --normclamp tanh  --maxcrop 1.0 --save_render --seed 11 --n_iter 1500  --learning_rate 0.0005 --normal_learning_rate 0.0005 --background 1 1 1 --frontview_center 0.5 0.6283
`

* cactus_texture.jpg
![image](https://user-images.githubusercontent.com/111476605/190899600-f5301783-74f1-4338-8b9f-6f52c38a0730.png)

* result
![image](https://user-images.githubusercontent.com/111476605/190899603-44d45f42-7d30-49e7-b59f-5ddcf24daac6.png)

Did I set the parameter wrong? Or is there something in main.py that needs to be modified? Or is it a randomness issue in optimization?

When --no_prompt is set to True and --image is set to image path string in main.py, the loss code corresponding to **'local to global'** and **'local to displacement'** in the paper is not understood. Should I change this part?
